src/deviation.jl

```diff
@@ -26,7 +39,12 @@ function countne(a::AbstractArray, b::AbstractArray)
     return c
 end
 
-# squared L2 distance
+
+"""
+    sqL2dist(a, b)
+
+Compute the squared L2 distance between two arrays: `∑|a-b|^2`.
```

Comment:
Will do, unless you think describing these with Julia code is better (see other comment). I guess this would be `sum(abs2(a - b))`.